src/main/java/org/apache/maven/plugins/clean/CleanMojo.java

```diff
@@ -161,6 +164,16 @@
     @Parameter( property = "maven.clean.excludeDefaultDirectories", defaultValue = "false" )
     private boolean excludeDefaultDirectories;
 
+    @Parameter( property = "maven.clean.fast", defaultValue = "false" )
+    private boolean fast;
+
+    @Parameter( property = "maven.clean.fastDir", defaultValue = "${maven.multiModuleProjectDirectory}/target/.clean" )
+    private File fastDir;
```

Comment:
Document the conditions for atomacy, I don't expect users to read source code to understand constant failures.